src/agenda/index.js

```diff
@@ -35,7 +31,7 @@ export default class AgendaView extends Component {
     /** Specify theme properties to override specific styles for calendar parts. Default = {} */
     theme: PropTypes.object,
     /** agenda container style */
-    style: viewPropTypes.style,
+    style: PropTypes.object,
```

Comment:
'style' prop type should be PropTypes.oneOfType([PropTypes.object, PropTypes.number, PropTypes.array]) as you can pass style inline as an object or array and you can pass it from a StyleSheet where it's just a number.

Reply:
Fixed